app/services/markdown_processor.rb

```diff
@@ -0,0 +1,80 @@
+# The purpose of this module is provide a common place for logic
```

Comment:
This looks like the sort of file we generally keep in `app/constants/` instead of `app/services` though in this specific case I think I actually prefer the current location.

Reply:
Yeah, definitely bridges between `app/constants` and `app/services`.

I want to "normalize" this so we can engage in conversations of what we expect.